source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -97,6 +108,21 @@ content: |
            
            BsonBinary dataKeyId = clientEncryption.createDataKey("aws", dataKeyOptions);
            String base64DataKeyId = Base64.getEncoder().encodeToString(dataKeyId.getData());
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+        
+           const encryption = new ClientEncryption(client, {
+               keyVaultNamespace,
+               kmsProviders
+           });
+           encryption.createDataKey("aws", {
+               masterKey: { 
+                 key: "<Master Key ARN>", // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
+                  region: "us-east-1"
```

Comment:
`region` should have a placeholder instead of a specific region

e.g.
```
{
  key: "<Master Key ARN>", // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
  region: "< Master Key AWS Region>", // e.g. "us-east-1"
}
```

Reply:
Fixed